### PR TITLE
Asn2gb

### PR DESCRIPTION
**Pull request by jamesabbott**
* [*] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [*] This PR adds a new recipe.
* [*] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This is a small NCBI utility for format conversion - one of a number they provide, including tbl2asn. The PR is therefore based on the tbl2asn recipe which uses binary builds of the utils. 

Contrary to the comments in the tbl2asn recipe, I believe the source code for these tools is available, but is part of the ncbi toolkit, which is a massive and complex build for a small utility, and I've not found the necessary configure options to build these utlitiies, so the binaries look a sensible option in this case.

Hopefully the use of binaries is acceptable in this case.

### CODE REVIEW

File: recipes/asn2gb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    home: https://www.ncbi.nlm.nih.gov/IEB/ToolBox/C_DOC/lxr/source/doc/asn2gb.txt
+    license: Public Domain
+    summary: asn2gb converts ASN1 format sequence records to Genbank format
+
+source:
+    fn: mac.asn2gb.gz # [osx]
+    url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/asn2gb/mac.asn2gb.gz # [osx]
+    sha256: e251014227d7cb9de7e33f2101b34c154d4ee40bc64b81d41dc4ed0fd5972d63 # [osx]
+    fn: linux.asn2gb.gz # [linux]
+    url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/asn2gb/linux.asn2gb.gz # [linux]
```
Comment (bgruening) [inline recipes/asn2gb/meta.yaml]: we only compile for linux64 ... so the selector can be just [linux]
Compiling is no option?

**Comment by jamesabbott** [comment]
The source is buried in the ncbi toolbox which is a huge build to get working (and I've not even managed to find the right incantation to make it build these conversion utilities). Since tbl2asn (another of these NCBI converters) has been added just as binaries I borrowed that recipe as a starting point. 

I'll take a look at what it would take to  get the ncbi toobox built...

**Comment by jamesabbott** [comment]
Actually, it doesn't look as as scary as I thought...

I'll have a go at building a recipe for the toolbox - it may duplicate standalone tools which have already been added to bioconda indivudually (i.e. tbl2asn)

**Comment by bgruening** [comment]
You call, I don't want to stop you :)

**Comment by jamesabbott** [comment]
I've hit  a brick wall building the ncbi-toolbox due to it's dependency on a c-shell configuration script. I've tried building a tcsh package for conda-forge but the build fails a bunch of tests. The tcsh project seems dead with the website gone and mailing list address undeliverable so without assistance from someone who can get to the bottom of the test failures I'm stuck.
Think I have to give up and revert to this binary version, if your happy to merge it.

**Comment by jamesabbott** [comment]
pinging @bgruening

**Comment by bgruening** [comment]
@jamesabbott thanks for trying that hard!

**Comment by bgruening** [comment]
Ok, this is a never ending story. The checksum does not match anymore and it was not uploaded to the archive. Can you mirror the binaries to cargo-port and we take these?

**Pull request merged**